software/spmd/instr_tests/branch_rv32/Makefile

```diff
@@ -1,6 +1,3 @@
-bsg_tiles_X= 4 
-bsg_tiles_Y= 4 
-
```

Comment:
In other places, you added bsg_tiles_X = 1 and bsg_tiles_Y = 1. Why did you add them in some places and remove them in others?

Reply:
I add it back for this. Most tests are meant to run only on a single tile.

Reply:
Is there a default bsg_tiles_X/Y?

Reply:
the default is the full array.